src/content/ccip/tutorials/usdc.mdx

```diff
@@ -0,0 +1,155 @@
+    1.  Fill in the arguments of the _**sendMessagePayLINK**_ function:
+
+        <br />
+
+        | Argument                   | Value and Description                                                                                                                                                                                                                                                                 |
+        | -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+        | \_destinationChainSelector | <CopyText text="12532609583862916517" code/> <br /> CCIP Chain identifier of the destination blockchain (_Polygon Mumbai_ in this example). You can find each chain selector on the [supported networks page](/ccip/supported-networks).                                              |
+        | \_receiver                 | Your receiver contract address at _Polygon Mumbai_. <br /> The destination contract address.                                                                                                                                                                                          |
+        | \_text                     | <CopyText text="Hello World!" code/><br />Any `string`                                                                                                                                                                                                                                |
+        | \_token                    | <CopyText text="0x5425890298aed601595a70AB815c96711a31Bc65" code/><br /> The _USDC_ contract address at the source chain (_Avalanche Fuji_ in this example). You can find all the addresses for each supported blockchain on the [supported networks page](/ccip/supported-networks). |
+        | \_amount                   | <CopyText text="1000000" code/> <br /> The token amount (_1 USDC_).                                                                                                                                                                                                                   |
+
+    1.  Click on `transact` and confirm the transaction on MetaMask.
+    1.  After the transaction is successful, record the transaction hash. Here is an [example](https://testnet.snowtrace.io/tx/0x6d54c98bb5b2892b78cbbc29967f4c0384d10dfc168755a4cf14489a02c11c65) of a transaction on _Avalanche Fuji_.
+
+    <Aside type="note">
+      During gas price spikes, your transaction might fail, requiring more than _0.5 LINK_ to proceed. If your
+      transaction fails, fund your contract with more _LINK_ tokens and try again.
```

Comment:
Does the CCIP Explorer or any error message indicate when the transaction fails due to this reason? If so, mention it here

Reply:
@thedriftofwords unfortunately not